lib/core/src/Cardano/Wallet/DB/Sqlite/CheckpointsOld.hs

```diff
@@ -175,27 +178,27 @@ import qualified Data.Map.Merge.Strict as Map
 import qualified Data.Map.Strict as Map
 
 {-------------------------------------------------------------------------------
-    Checkpoints Store
+    WalletState Store
 -------------------------------------------------------------------------------}
--- | Store for 'Checkpoints' of multiple different wallets.
-mkStoreWalletsCheckpoints
+-- | Store for 'WalletState' of multiple different wallets.
+mkStoreWallets
     :: forall s key. (PersistAddressBook s, key ~ W.WalletId)
     => Store (SqlPersistT IO)
-        (DeltaMap key (DeltaCheckpoints (W.Wallet s)))
-mkStoreWalletsCheckpoints = Store{loadS=load,writeS=write,updateS=update}
+        (DeltaMap key (DeltaWalletState s))
+mkStoreWallets = Store{loadS=load,writeS=write,updateS=update}
   where
     write = error "mkStoreWalletsCheckpoints: not implemented"
 
     update _ (Insert wid a) =
-        writeS (mkStoreCheckpoints wid) a
+        writeS (mkStoreWallet wid) a
     update _ (Delete wid) = do
         -- FIXME LATER during ADP-1043:
         --  Deleting an entry in the Checkpoint table
         --  will trigger a delete cascade. We want this cascade
         --  to be explicit in our code.
         deleteWhere [CheckpointWalletId ==. wid]
     update _ (Adjust wid da) =
-        updateS (mkStoreCheckpoints wid) undefined da
+        updateS (mkStoreWallet wid) undefined da
         -- FIXME LATER during ADP-1043:
```

Comment:
aha, there is plan to replace it properly :+1:

Reply:
Yes. I'll remove `undefined` soon.